C/Cpp_include/c4Query.hh

```diff
@@ -89,28 +89,28 @@ struct C4Query final
     /// auto e = query.run();
     /// while (e.next()) { ... }
     /// ```
-    Enumerator run(const C4QueryOptions* C4NULLABLE opt = nullptr, slice params = fleece::nullslice);
+    Enumerator run(slice params = fleece::nullslice);
```

Comment:
If nobody is using this C4QueryOption, we may remove this from the API as well.

```
/** Options for running queries. */
typedef struct {
    bool rankFullText_DEPRECATED;      ///< Ignored; use the `rank()` query function instead.
} C4QueryOptions;
```

Reply:
Should I create a separate ticket for that?

Reply:
Yes

Reply:
https://issues.couchbase.com/browse/CBL-4331